packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -614,7 +614,7 @@ export interface FieldPathWithCount {
 function isField1UnderField2(field1: FlexTreeField, field2: FlexTreeField): boolean {
 	let parentField = field1.parent?.parentField?.parent;
 	while (parentField !== undefined) {
-		if (parentField.isSameAs(field2)) {
```

Comment:
If we expose access tot he field anchor, this test could compare them. We should have a util for that

Reply:
Er, why?

Reply:
apparently, these fuzz tests need to test `isField1UnderField2`.  SInce using === is not guaranteed to produce correct results in general for comparing fields due to the issues I pointed out in the comment I posted at the top level (copy link to it isn't working for me), this code would need to use something else. Comparing field anchors paths has the desired semantics since the goal here is to check if the paths are the same (meaning its the same logical field). Could could also just inline what ever isSameAs does here .

Reply:
Thank you for the explanation :)

I opted to simply preserve the previous behavior as you suggested.

Reply:
After we get the flex layer cleaned up, we can revisit how we want to compare the identity of logical fields